src/main/java/nl/cherement/jak/repository/ColumnRepository.java

```diff
@@ -0,0 +1,17 @@
+package nl.cherement.jak.repository;
+
+import nl.cherement.jak.entity.ColumnEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface ColumnRepository
+        extends JpaRepository<ColumnEntity, Long> {
+
+    @Query(value = "SELECT c FROM ColumnEntity c WHERE c.boardId = :id")
+    List<ColumnEntity> getByBoardId(@Param("id") Long id);
+}
```

Comment:
De query is niet vereist bij het opvragen van de data.
Dit zit standaard geintregeerd in https://docs.spring.io/spring-data/jpa/docs/1.5.0.RELEASE/reference/html/jpa.repositories.html

Reply:
Nice. Moeten we van de week even fixen dan